### PR TITLE
Fixed data processing step

### PR DESCRIPTION
**Pull request by baimamboukar**

### CODE REVIEW

File: data/nuget.csv [hunk 1]
```diff
@@ -16064,8 +16064,8 @@ version,base version,scenario,test case,timestamp,duration,base duration,relativ
 8.0.100-rc.1.23410.12,7.0.100,cold,Orleans-eda972a,2023-08-11T00:12:02.9772229Z,33.2036814,15.0829142,220.14102155404424
 8.0.100-rc.1.23410.12,7.0.100,force,Orleans-eda972a,2023-08-11T00:12:02.9772229Z,7.5395383,5.5924925,134.81534932769245
 8.0.100-rc.1.23410.12,7.0.100,noop,Orleans-eda972a,2023-08-11T00:12:02.9772229Z,5.5363815,3.9919522,138.68857197237983
-<<<<<<< HEAD:data/nuget.csv,,,,,,,
-=======,,,,,,,
+<<<<<<< HEAD:data/nuget.csv
+=======
```
Comment (marcin-krystianc) [inline data/nuget.csv:16068]:
```suggestion
```

Comment (marcin-krystianc) [reply data/nuget.csv:16068]: These lines need to be removed 😄

Comment (baimamboukar) [reply data/nuget.csv]: Okay @marcin-krystianc ..... Sorry for this... I was sure I fixed everything 😅

Comment (baimamboukar) [reply data/nuget.csv]: I did fixed it..... It should be OK now

**Review by marcin-krystianc — APPROVED** [review]

**Pull request merged**